packages/design-docs/src/status.json

```diff
@@ -0,0 +1,699 @@
+{
+  "accordion": {
```

Comment:
This create an issue: it duplicates the status of the sync. it will makes it too hard to update.